fullstack/README.md

```diff
@@ -0,0 +1,64 @@
+This is a coding exercise to help assess candidates looking to join our team at Condé Nast International as FullStack Developers.
+
+The test is intended to be completed at home in no more than 1-2 hours. It is not timed, so if you want to take more time to showcase your coding skills and ability you may do so (note this is not mandatory, nor expected).
+
+If you haven't finished in that time, you can send us your progress - we don't want you to spend days on this. We understand that your time is valuable, and appreciate any time that you contribute towards building a strong team here.
+
+If you **really cannot spare the time**, you may want to take a look at Option 2.
+
+## Option 1
+
+Build a JavaScript application that shows the latest news from the United Kingdom, using the [newsapi.org](https://newsapi.org) service.
+
+Running the app we should be able to see the latest **UK news** and give the ability to the user to filter them by categories. Given the fact that this is a FullStack challenge, we are expecting a Frontend application and a Backend application that communicate with each other.
+
+A FullStack developer is someone with familiarity in each layer; you might be stronger on the frontend, or vice-versa. Feel free to focus more on what you know best, be creative and show us your skills!
+
+### What we are looking for
+
+* We anticipate that the website may have future requirements, so you should aim for maintainability with your solution
+* The JavaScript ecosystem changes quickly, so we like to see knowledge of JavaScript fundamentals
+* We want to discover how you write code day-to-day (from the folder structure, testing, naming point of view)
+* We don't want you to spend hours tweaking every pixel. Keep it simple.
+* Keep it simple (again :) )
+
+### Submission
+
+You can submit your code however you want (github/bitbucket/gitlab repo - git bundle - zip file)- send the details to your point of contact.
```

Comment:
Perhaps we should request that they *not* submit a pull request? Actually maybe we could make a new repo to remove the word 'test' from the name (maybe more like cni-code-exercise) and remove the ability to submit PRs.

Reply:
My suggestion was more about the fact that some candidates don't want to explicitly show that they are looking for a job. If we allow PR, people in the candidate's circle can see the activity.